tdrs-frontend/src/components/SiteMap/SiteMap.jsx

```diff
@@ -27,6 +31,9 @@ const SiteMap = ({ user }) => {
         target="_blank"
       />
       {userIsApproved && <SiteMap.Link text="Data Files" link="/data-files" />}
```

Comment:
```suggestion
      {userIsApproved && <SiteMap.Link text="TANF Data Files" link="/data-files" />}
```

@jtimpe is this the only place we need to make this change to ensure the sitemap label for TANF Data Files is correct? cc: @ttran-hub 

![Screenshot 2025-02-25 141716](https://github.com/user-attachments/assets/066b4eff-4069-45f5-a048-3dd3da1c847d)

Reply:
oops, missed that. thank you! i believe that is the only place